projects/README.md

```diff
@@ -0,0 +1,7 @@
+## Managing Projects 
+
+Repositories aren't just a place to upload code - they're a place to collaborate and work together. Depending on how you choose to manage projects, that process should be as transparent and integrated as possible. 
+
+[Surveys show](https://m.signalvnoise.com/the-3-most-effective-ways-to-build-trust-as-a-leader/) that trust not only is built by doing good - but by communicating the intent behind those actions. 
+
```

Comment:
I agree that good communication is important. We use it to apply to our own words so that others can understand and collaborate.